src/Marten/Schema/BulkLoading/BulkLoader.cs

```diff
@@ -121,6 +121,8 @@ private void load(ITenant tenant, ISerializer serializer, NpgsqlConnection conn,
                     _transferData(document, _mapping.AliasFor(document.GetType()), serializer, writer, textWriter, tenant.TenantId);
                     textWriter.Clear();
                 }
+                
+                writer.Complete();
```

Comment:
This is new in npgsql4?

Reply:
I believe so. @jacobpovar ?

Reply:
Correct. 

From [npgsql 4 breaking changes](http://www.npgsql.org/doc/release-notes/4.0.html):

> You must now call NpgsqlBinaryImporter.Complete() to save your imported data; not doing so will roll the operation back. 

> 

>